packages/cip30/test/mockCardanoWallet.test.ts

```diff
@@ -0,0 +1,177 @@
+/* eslint-disable sonarjs/no-duplicate-string */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { MockCardanoWallet } from './mockCardanoWallet';
+import { RequestAccess } from '../src/BaseCardanoWallet';
+import { CardanoWalletApi } from '../src/CardanoWalletApi';
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { mocks } from 'mock-browser';
+
+const window = mocks.MockBrowser.createWindow();
+
+const props = { name: 'test-wallet', version: '0.0.0-test' };
+const _api = <CardanoWalletApi>{
+  getUtxos: async (_amount) => [
+    [
+      { txId: '123456', index: 0 },
+      { address: 'asdf', value: { coins: 100, assets: {} } }
+    ]
+  ],
+  getBalance: async () => '100',
+  getUsedAddresses: async () => ['used-address-1', 'used-address-2', 'used-address-3'],
+  getUnusedAddresses: async () => ['unused-address-1', 'unused-address-2', 'unused-address-3'],
+  getChangeAddress: async () => 'change-address',
+  getRewardAddresses: async () => ['reward-address-1', 'reward-address-2'],
+  signTx: async (_tx) => 'signedTransaction',
+  signData: async (_addr, _sig) => 'signedData',
+  submitTx: async (_tx) => 'transactionId'
+};
+const requestAccess: RequestAccess = async () => true;
+const options = { persistAllowList: false };
```

Comment:
I'm OK with us not testing the persisted allow list until we have finalised the design.